extension/hotfix-intermediate-2018.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no console */
+/* global ExtensionAPI */
+
+// This pref is being introduced by Bug 1954934.
+const PREF_LAST_SIGNATURE_CHECKPOINT = "extensions.signatureCheckpoint";
+
+this.hotfixIntermediateCert = class extends ExtensionAPI {
+  onStartup() {
+    const fxMajorVersion = parseInt(Services.appinfo.version.split(".")[0], 10);
+    if (fxMajorVersion >= 137) {
+      // Return earlier on Firefox versions where this hotfix is not needed.
+      return;
```

Comment:
should we log an info message to mention that we are returning early?

Reply:
No, it would be logspam.

Reply:
We can gate that behind `extensions.logging.enabled`. That'd be useful for QA.